tomcat/src/main/java/com/techcourse/http/request/HttpRequest.java

```diff
@@ -46,66 +89,70 @@ private static void validateNullRequestLine(final String requestLine) {
     }
 
     private static void validateRequestLineFormat(final String[] requestLineValues) {
-        if (requestLineValues.length != 3) {
+        if (requestLineValues.length != REQUEST_LINE_PARTS) {
             throw new UncheckedServletException("올바르지 않은 요청 형식입니다.");
         }
     }
 
     private static String extractPath(final String requestUri) {
-        int queryStringStartIndex = findSingleDelimiterIndex(requestUri, "?");
+        int queryStringDelimiterIndex = findQueryStringDelimiterIndex(requestUri);
 
-        if (queryStringStartIndex != -1) {
-            return requestUri.substring(0, queryStringStartIndex);
+        if (queryStringDelimiterIndex != NOT_FOUND_INDEX) {
+            return requestUri.substring(0, queryStringDelimiterIndex);
         }
 
         return requestUri;
     }
 
     private static String extractQueryString(final String requestUri) {
-        int queryStringStartIndex = findSingleDelimiterIndex(requestUri, "?");
+        int queryStringDelimiterIndex = findQueryStringDelimiterIndex(requestUri);
 
-        if (queryStringStartIndex != -1) {
-            return requestUri.substring(queryStringStartIndex + 1);
+        if (queryStringDelimiterIndex != NOT_FOUND_INDEX) {
+            return requestUri.substring(queryStringDelimiterIndex + 1);
         }
 
-        return "";
+        return EMPTY;
     }
 
-    private static ContentType extractContentType(final String path) {
-        Objects.requireNonNull(path);
-        int delimiterIndex = findSingleDelimiterIndex(path, ".");
+    private static int findQueryStringDelimiterIndex(final String uri) {
+        int firstIndex = uri.indexOf(QUERY_STRING_DELIMITER);
+        int lastIndex = uri.lastIndexOf(QUERY_STRING_DELIMITER);
 
-        if (delimiterIndex != -1) {
-            String extension = path.substring(delimiterIndex + 1);
-            return ContentType.from(extension);
+        if (firstIndex != NOT_FOUND_INDEX && firstIndex != lastIndex) {
+            throw new UncheckedServletException("잘못된 URI 형식: ?가 여러 번 포함되었습니다 → " + uri);
         }
-        return ContentType.TEXT_HTML;
+
+        return firstIndex;
     }
 
-    private static int findSingleDelimiterIndex(String uri, String delimiter) {
-        int firstIndex = uri.indexOf(delimiter);
-        int lastIndex = uri.lastIndexOf(delimiter);
+    private static ContentType extractContentType(final String path) {
+        Objects.requireNonNull(path);
+        int extensionDelimiterIndex = path.lastIndexOf(EXTENSION_DELIMITER);
 
-        if (firstIndex != -1 && firstIndex != lastIndex) {
-            throw new UncheckedServletException("잘못된 URI 형식: " + delimiter + " 가 여러 번 포함되었습니다 → " + uri);
+        if (extensionDelimiterIndex != NOT_FOUND_INDEX) {
```

Comment:
그냥 참고(개인적인 의견)

`extensionDelimiterIndex != NOT_FOUND_INDEX`가 `extensionDelimiterIndex != -1`보다 가독성이 좋은지 잘 모르겠어요. 차라리 메서드로 표현하는건 어떤가요?
```suggestion
        if (hasExtension(extensionDelimiterIndex)) {
        ...
        }
        
        private boolean hasExtension(int delimiterIndex) {
            return delimiterIndex != NOT_FOUND_INDEX;
        }
```